src/jit/lclvars.cpp

```diff
@@ -77,6 +77,9 @@ void                Compiler::lvaInit()
     lvaSIMDInitTempVarNum = BAD_VAR_NUM;
 #endif // FEATURE_SIMD
     lvaCurEpoch = 0;
+#ifdef FEATURE_MULTIREG_STRUCT_ARGS
```

Comment:
Here too the #ifdef has to be FEATURE_UNIX_AMD64_STRUCT_PASSING? as it has nothing to do with multireg or struct passing but something to do with Unix Amd64 ABI.